docs/hoon/library/1a.md

```diff
@@ -64,7 +64,469 @@ Source
 
     ++  dec                                                 ::  decrement
       ~/  %dec
-      |=  a/@
+      |=  a=@
+      ~_  leaf+"decrement-underflow"
+      ?<  =(0 a)
+      =+  b=0
+      |-  ^-  @
+      ?:  =(a +(b))  b
+      $(b +(b))
+
+Examples
+--------
+
+    > (dec 7)
+    6
+    > (dec 0)
+    ! decrement-underflow
+    ! exit
+
+
+
+***
+### `++div`
+
+Divide
+
+Computes `a` divided by `b`.
+
+Accepts
+-------
+
+`a` is an atom.
+
+`b` is an atom.
+
+Produces
+--------
+
+An atom.
+
+Source
+------
+
+    ++  div                                                 ::  divide
+      ~/  %div
+      |:  [a=`@`1 b=`@`1]
+      :>  quotient
```

Comment:
We need to figure out whether to include doccord lines in the `hoon.hoon` code or not.  Check with the infrastructure team about whether to include these.  Either include them all, or get rid of the `:>  quotient` here.

Reply:
Doccords are going away, to be replaced with leading comments, like this:
```
::  +some-arm: one-line description, all lower-case, no trailing period
::
++  some-arm  ...
::  +another-arm: one-line description
::
::    Long-form comments have four leading spaces and
::    use complete sentences. One blank comment line follows.
::
++  another-arm
  ::  some-face: a face declaration
  ::
  =/  some-face=@  42
  ...
```

Reply:
For more details on this style, see the [ford-turbo branch](https://github.com/urbit/arvo/blob/ford-turbo/lib/ford-turbo.hoon).